### PR TITLE
Try to fill 'userInfo' structure if 'idToken' is not defined in case of brokered authentication

### PR DESCRIPTION
**Pull request by matrosov-nikita**
Related issue: https://github.com/AzureAD/azure-activedirectory-library-for-cordova/issues/127

### CODE REVIEW

**Comment by vladimir-kotikov** [comment]
LGTM. @daserge?

File: www/AuthenticationResult.js [hunk 1]
```diff
@@ -17,7 +17,16 @@ function AuthenticationResult(authResult) {
...
-    this.userInfo = authResult.idToken ? UserInfo.fromJWT(authResult.idToken) : null;
+    var jwtToken = authResult.idToken || authResult.accessToken;
+    this.userInfo = null;
+
+    if (jwtToken) {
+        this.userInfo = UserInfo.fromJWT(jwtToken);
+    }
+
+    if (!this.userInfo) {
+        this.userInfo = new UserInfo(authResult.userInfo);
```
Comment (daserge) [inline www/AuthenticationResult.js]: Can `authResult.userInfo` be null? @matrosov-nikita

Comment (matrosov-nikita) [inline www/AuthenticationResult.js]: @daserge, yes, it's verified [there](https://github.com/AzureAD/azure-activedirectory-library-for-cordova/blob/master/www/UserInfo.js#L12)

Comment (daserge) [inline www/AuthenticationResult.js]: Got it, thanks!

**Comment by daserge** [comment]
LGTM 👍

**Pull request merged**